src/stars.css

```diff
@@ -0,0 +1,88 @@
+.stars {
+  display: flex;
+  width: 96px;
```

Comment:
Remove this `width`, this should fix the tests
```suggestion
```

Reply:
Not done yet, try running the tests without this using `npm run test:only`. Fixed the tests on my machine.